README.md

```diff
@@ -39,15 +39,18 @@ IRC](https://freenode.net/kb/answer/chat).
 Nmstate uses `tox` to run unit tests and linters. Since Nmstate uses the binary
 module PyGObject it also requires the build dependencies for it.
 
-### RHEL 7.6
```

Comment:
Why would you like to remove the instructions for RHEL? IMHO it makes sense to have them as well.

Reply:
I just thought that instructing RHEL users to use `epel` and `pip` is not a good move (or supported), but if you think it makes sense, then it is fine by me.

Reply:
Using EPEL and pip is the pragmatic way to be able to develop on RHEL and I would like to be be able to do it if necessary, so please keep it.

Reply:
Ok, I will keep it.
But for the record, based on my experience, the pragmatic way is to use a container instead. We should probably add documentation for it as well.